src/lib.rs

```diff
@@ -0,0 +1,6 @@
+#[macro_use]
+extern crate lazy_static;
```

Comment:
`extern crate` statements aren't required in the 2018 edition.

Reply:
For some reason it doesn't work for me without this lines. I can investigate it, though I could be not the only one for whom this doesn't work, so I would leave it as is